File: src/my_project/easy_problems/from201to250/fizz_buzz.py
from typing import List 

class Solution:
    def fizzBuzz(self, n: int) -> List[str]:

        solution = [str(i+1) for i in range(n)]
        for i in range(n):
            if (i+1)% 3 == 0 and (i+1)%5 == 0:
                solution[i] = "FizzBuzz"
            elif (i+1) % 3 == 0:
                solution[i] = "Fizz"
            elif (i+1) % 5 == 0:
                solution[i] = "Buzz"

        return solution 

solution = Solution()

print(solution.fizzBuzz(n = 3))

print(solution.fizzBuzz(n = 5))

print(solution.fizzBuzz( n = 15))

File: my_project_typescript/easy_problems_typescript/from101to150/fizzbuzz.ts
function fizzBuzz(n: number): string[] {
    let solution: Array<string> = []
    for (let i = 1; i <= n; i++){
        solution.push(i.toString())
    }

    for (let i = 0; i < n; i++){
        if ((i+1)%3 === 0 && (i+1)%5 ===0){
            solution[i] = "FizzBuzz"
        }
        else if ( (i+1)%3 === 0){
            solution[i] = "Fizz"
        }
        else if ( (i+1)%5 === 0){
            solution[i] = "Buzz"
        }
    }
    return solution
};

console.log(fizzBuzz(3))

console.log(fizzBuzz(5))

console.log(fizzBuzz(15))
